programmers-coding-test/kimjiwon/greedy/큰수의법칙.js

```diff
@@ -0,0 +1,30 @@
+function bigNumberRule(firstLine) {
+  const rule = firstLine.split(" ");
```

Comment:
따로 받아서 사용하는게 가독성에 좋을 듯 싶다.
```suggestion
  const [ n, m, k ] = firstLine.split(" ");
```